pkg/kotsadmsnapshot/backup.go

```diff
@@ -194,12 +193,12 @@ func CreateApplicationBackup(ctx context.Context, a *apptypes.App, isScheduled b
 		logger.Error(errors.Wrap(err, "failed to exclude shutdown pods from backup"))
 	}
 
-	backup, err := veleroClient.Backups(kotsadmVeleroBackendStorageLocation.Namespace).Create(ctx, veleroBackup, metav1.CreateOptions{})
+	err = veleroClient.Create(ctx, veleroBackup)
```

Comment:
Namespace is already set above - https://github.com/replicatedhq/kots/blob/a422a554bd22d7b1e7e0feed4f76e33d77f0fc57/pkg/kotsadmsnapshot/backup.go#L153